src/components/common/viewHumanReadableModal/ViewHRModal.tsx

```diff
@@ -42,7 +50,7 @@ export default function ViewHRModal(props: ModalProps) {
 
   useEffect(() => {
     getHumanReadable(props.measureId);
-  }, [props.measureId]);
+  }, [props.measureId, measure]);
```

Comment:
did props.measureId not work? its the only dependency along with getHumanReadable of this useEffect.